server/planning/events/module.py

```diff
@@ -28,3 +30,10 @@
     ),
     elastic=ElasticResourceConfig(),
 )
+
+events_history_resource_config = ResourceConfig(
+    name="events_history",
+    data_class=EventsHistoryResourceModel,
+    service=EventsHistoryAsyncService,
+    rest_endpoints=RestEndpointConfig(resource_methods=["GET"], item_methods=["GET"]),
```

Comment:
If we're going to enable the REST API endpoints, then we should probably add `internal_resource = True` to the `EventsHistoryResource` and `PlanningHistoryResource` legacy resources, otherwise they'll both be trying to register the same endpoint with Flask